src/app/components/hmcts-primary-navigation/hmcts-primary-navigation.component.html

```diff
@@ -10,7 +10,7 @@
         </ul>
       </nav>
     </div>
-    <div class="hmcts-primary-navigation__search">
+    <div class="hmcts-primary-navigation__search" *ngIf="items?.length">
```

Comment:
because if there are not nav items the search shouldn't be displayed... unfortunately the search is not done programmatically so i found that this short cut works